virt_lightning/configuration.py

```diff
@@ -61,7 +59,11 @@ def __init__(self):
             self.load_file(DEFAULT_CONFIGFILE.expanduser())
 
     def __get(self, key):
-        return self.data.get("main", key)
+        try:
```

Comment:
I'm not totally sure we need this construction. I would put everything in `main`. Overall I feel like the use of the INI format is unfortunate :-).